pal/screen-adapter/web/screen-adapter.ts

```diff
@@ -504,7 +548,8 @@ class ScreenAdapter extends EventTarget {
         const height = window.innerHeight;
         const isBrowserLandscape = width > height;
         this.isFrameRotated = systemInfo.isMobile
-            && ((isBrowserLandscape && orientation === Orientation.PORTRAIT) || (!isBrowserLandscape && orientation === Orientation.LANDSCAPE));
+            && ((isBrowserLandscape && orientation === Orientation.PORTRAIT) || (!isBrowserLandscape
+            && (orientation === Orientation.LANDSCAPE || orientation === Orientation.LANDSCAPE_LEFT || orientation === Orientation.LANDSCAPE_RIGHT)));
```

Comment:
This `&&` is different used with `!isBrowserLandscape`, so please format the indent to make it more clear.